src/gui/base/List.ts

```diff
@@ -667,7 +666,7 @@ export class List<ElementType extends ListElement, RowType extends VirtualRow<El
 			shiftPressed = false
 		}
 
-		if (shiftPressed && this.lastMultiSelectWasKeyUp === true && this.selectedEntities.length > 1) {
+		if (shiftPressed && this.lastMultiSelectWasKeyUp && this.selectedEntities.length > 1) {
```

Comment:
some random changes?

Reply:
whoops. I haphazardly pulled this out of my indexing investigation branch